spec/fixtures/custom_nginx.template

```diff
@@ -361,6 +376,13 @@ http {
             }
         }
 
+        location ~ "^/cache/(?<n>\d+)$" {
+          content_by_lua_block {
+            local mu = require "spec.fixtures.mock_upstream"
+            return mu.send_default_json_response({}, {["Cache-Control"] = "public, max-age=" .. ngx.var.n})
+          }
```

Comment:
style: this file follows 4 space indentation rule - mind updating this block?

Reply:
Ooops. Done, thanks Thibault!